modules/statics/test/unit/coins.ts

```diff
@@ -682,3 +682,28 @@ describe('Bulk Transaction Features', () => {
     });
   });
 });
+
+describe('Eip1559 coins', () => {
+  const eip1559Coins = [
+    'avaxc',
+    'eth',
+    'hteth:bgerchv2',
+    'tavaxc',
```

Comment:
nit: can we reorder these so `avaxc` and `tavaxc` are next to each other? prefer all the mainnet coins to be next to their testnet coins